<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVaccination</class>
 <widget class="QWidget" name="AddVaccination">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbl_topBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/images/topBar.png</pixmap>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>375</width>
     <height>476</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="lbl_dateOfTest">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>40</y>
       <width>331</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="text">
      <string>Date of vaccination (DD/MM/YY)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_YY">
     <property name="geometry">
      <rect>
       <x>142</x>
       <y>70</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>21</number>
     </property>
     <property name="maximum">
      <number>23</number>
     </property>
     <property name="value">
      <number>21</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_DD">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>31</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>265</y>
       <width>330</width>
       <height>50</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_MM">
     <property name="geometry">
      <rect>
       <x>81</x>
       <y>70</y>
       <width>61</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>12</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>170</y>
       <width>330</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>330</width>
       <height>41</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Pfizer-BioNTech</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GlaxoSmithKline</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Novavax</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>136</y>
       <width>133</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Manufacturer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>320</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>14</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;Arial&quot;;</string>
       </property>
       <property name="text">
        <string>New vaccine data added</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbl_DoV">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Arial&quot;;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Date vaccinated:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl_displayVaxDate">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>[Test date]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbl_dM">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>10</pointsize>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Arial&quot;;</string>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string>Manufacturer: </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="lbl_displayVaxManuf">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>[Test result]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="text">
          <string>BACK TO HOMEPAGE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableView" name="tableView_test">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>331</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_visualisingUserPID">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Hidden page for visualising db</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lbl_QRRequests_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-2</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-size:20px;
font-family:arial;</string>
   </property>
   <property name="text">
    <string>Add Vaccination</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_backToAdminHome">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-2</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/backArrow2.png);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
